_posts/2025-05-14-#32317.md

```diff
@@ -38,13 +57,13 @@ validate a transaction by providing the validation function the specific UTXTOs
 2. Why is carving out the new `SpendBlock()` function from `ConnectBlock()` helpful for this PR? How
    would you compare the purpose of the two functions?
 
-4. Do you see another benefit of this decoupling, besides allowing kernel usage without a UTXO set?
```

Comment:
fwiw I always just use `1.` and markdown numbers them automatically

Reply:
Yeah it's my preferred approach too but my markdown plugin won't let me disable just the automatic numbering, so sometimes I give in and let it do its bloody thing.